There is one failing check, the err is:
```
13-slim: Pulling from library/debian
received unexpected HTTP status: 502 Bad Gateway
Error: Process completed with exit code 1.
```
some network hiccup, but i can't retry the job, so hopefully some of the maintainers will get to that 😊

--- a/library/azul-zulu
+++ b/library/azul-zulu
@@ -7,7 +7,7 @@ Maintainers: Azul Devops Team <devops@azul.com> (@azul-publisher),
              Geertjan Wielenga <gwielenga@azul.com> (@geertjanw)
 GitRepo: https://github.com/AzulSystems/azul-zulu-images.git
 GitFetch: refs/heads/main
-GitCommit: eaa5481239a895ec585325e02be86f4720582abe
+GitCommit: c3c5ad0d77beee6063b4e1496fe32f417af274ed
 
 
 Tags: 8.94-8.0.492-jdk-almalinux10, 8-almalinux10, 8-jdk-almalinux, 8-jdk-almalinux10

